web/apps/serve.go

```diff
@@ -133,6 +133,12 @@ func ServeAppFile(c echo.Context, i *instance.Instance, fs AppFileServer, app *a
 		return err
 	}
 	defer content.Close()
+
+	ext := path.Ext(file)
+	if ext != ".html" && ext != ".htm" {
+		return fs.ServeFileContent(c.Response(), c.Request(), modtime, slug, route.Folder, file)
+	}
```

Comment:
My bad, did expect to apply this in this PR. It is not relevant anyway like you said... removed it f0660a4292d05eb3602ab60dca22e03a312aef94